specification.markdown

```diff
@@ -328,6 +329,20 @@ with a length greater than 4 octets. All EBML Elements within the EBML
 Header MUST NOT utilize any Element Data Size with a length greater
 than 4 octets.
 
+### EBML Body
+
+All data of an EBML Document following the EBML Header may be considered
+the EBML Body. The end of the EBML Body, as well as the end of the EBML
+Document that it is contained within, may be considered as whichever
+comes first: the beginning of a new level 0 EBML Header or the end of
+the file. The EBML Body MAY only consist of EBML Elements and MUST NOT
+contain any data that is not part of an EBML Element. Although the EBML
+specification itself defines precisely what EBML Elements are to be
+used within the EBML Header, the EBML specification does not name or
+define what EBML Elements are to be used within the EBML Body. The
+definition of what EBML Elements are to be used within the EBML Body is
+defined by an EBML Schema.
```

Comment:
EBML Schema is to be defined. I think to define it I'll start by extending the section called `EBML Template` to define the element descriptors more specifically. However to define what a proper expression of an `EBML Schema` is goes back to some open listserv discussions about specdata.xml and how to express a machine-readable `EBML Schema`.